src/PrestaShopBundle/Controller/Admin/Configure/AdvancedParameters/EmployeeController.php

```diff
@@ -476,6 +478,20 @@ protected function getEmployeeFormHandler()
     protected function getErrorMessages(Exception $e)
     {
         return [
+            PasswordConstraintException::class => [
+                PasswordConstraintException::INVALID_LENGTH => $this->trans(
+                    'Password length must be between %min% and %max% characters.',
```

Comment:
```suggestion
                    'Password must be between %d and %d characters long.',
```
I found a preexisting similar string. Would this one work?